src/scss/trumps/_utilities.scss

```diff
@@ -275,3 +275,32 @@
   outline: none;
   box-shadow: 0 0 0 2px $color-focus-outline;
 }
+
+
+//
+// Text Utilities
```

Comment:
There are already some "Typographic Helpers" in a section above, is it worth pulling the two to sit together?